README.md

```diff
@@ -1,18 +1,29 @@
-## openHAB 2 Add-ons
+# openHAB 2 Add-ons
 
-This repository contains add-ons that are implemented using the new [Eclipse SmartHome APIs](https://www.eclipse.org/smarthome/documentation/development/bindings/how-to.html) of openHAB 2.
+This repository contains the official set of add-ons that are implemented on top of openHAB 2 Core APIs.
+Add-ons that got accepted in here will be maintained (e.g. adapted to new core APIs)
+by the [openHAB 2 maintainers](https://github.com/orgs/openhab/teams/2-x-add-ons-maintainers).
 
-Note that all information about openHAB itself, the IDE setup and the contribution processes can be found in the [openhab-distro](https://github.com/openhab/openhab-distro) project, so please go there for any further details!
+To get started with binding development, follow our guidelines and tutorials over at https://www.openhab.org/docs/developer/.
```

Comment:
URL without link-syntax.

Reply:
Yes they are on purpose, as I want to raise awareness of how our link structure looks like. 

Especially for the other-addon-repositories, so that a reader can draw conclusions on where to look for more addon repos.

Reply:
Thanks for clarification. :+1: